pom.xml

```diff
@@ -3,10 +3,10 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>spark-connector</artifactId>
-    <version>1.2.0</version>
+    <version>1.2.1</version>
     <name>The Spark Exasol Connector</name>
     <description>A connector for Apache Spark to access Exasol</description>
-    <url>https://github.com/exasol/spark-connector</url>
+    <url>https://github.com/exasol/spark-connector/</url>
     <properties>
         <java.version>1.8</java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
Delete `<properties>` and `<licenses>` and `<developers>` and `<scm>`, they are already defined in the parent pom.